app/degreePlanner/page.jsx

```diff
@@ -1,61 +1,8 @@
-"use client"
+import { getDegrees, getMajors } from "../api/utils/degrees";
 import "./degreePlanner.css";
-import { useState } from "react";
-import { Planner } from "./planner";
+import FormDetails from "./formDetails";
 
```

Comment:
What I have done here is change the page from a client component to a server component, and instead embedded a client component inside the server component.

Think of a client component as literally creating responsiveness on client. Everything that is dynamic is on the client.

A server component instead is something that is _completely pre-rendered and sent to the client._. Whereas a client component obv changes on the client and is not completely pre-rendered. 

This also means that github things I deleted and added a bunch of lines but I didn't I extracted anything that uses a react state into a client component `<FormDetails />`.  Use state all happens as client reactivity.

Reply:
If you don't understand this @jennifermtan defo look into it.

Reply:
Looks good from a high level perspective. I will definitely look into it more when I touch on this in the next milestones.